t/Anime.t

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Anime)],
+    'anime pokemon' => test_spice(
+        '/js/spice/anime/pokemon',
+        call_type => 'include',
+        caller => 'DDG::Spice::Anime'
+    ),
+    'anime Medabots' => test_spice(
```

Comment:
Please make sure your tests at least cover the example queries you've listed.

It would be great to add some example queries that shouldn't trigger this.

ie. `anime movies coming soon => undef;`